From: John Paul Adrian Glaubitz <glaubitz@physik.fu-berlin.de>
Date: Fri, 27 Dec 2019 12:22:20 +0100
Subject: Define unknown host CPUs as CPU_unknown.

---
 src/include/sysdeps.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/include/sysdeps.h b/src/include/sysdeps.h
index 607ff32..235183e 100644
--- a/src/include/sysdeps.h
+++ b/src/include/sysdeps.h
@@ -50,7 +50,7 @@ using namespace std;
 #elif defined(__powerpc__) || defined(__ppc__) || defined(_M_PPC)
 #define CPU_powerpc 1
 #else
-#error unrecognized CPU type
+#define CPU_unknown 1
 #endif
 
 #ifdef _WIN32
